github-maintenance/labelRepo.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+#ADD LABELS TO REPOS
+
+#PROVIDE THE LOCATION TO THE FILE WITH LIST OF REPOS NAMES
+
+repoList="/<PATH>/<FILE NAME>.txt"
+
+FILELINE=`cat $repoList`
+
+echo START
+
+for LINE in $FILELINE ; do
+
+#adding type
+curl --include --request POST --data '{"name":"Type/Bug","color":"1d76db"}' "https://api.github.com/repos/$LINE/labels?access_token=<TOKEN>"
```

Comment:
It is better if you can specify <TOKEN> as a variable. This can be taken as an input parameter. See [here](http://how-to.wikia.com/wiki/How_to_read_command_line_arguments_in_a_bash_script).

```
TOKEN=$1    # first input parameter
curl --include --request POST --data '{"name":"Type/Bug","color":"1d76db"}' "https://api.github.com/repos/$LINE/labels?access_token=$(TOKEN)"

```